packages/core-db/test/app/ethereum/contracts/TestToken.json

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
Oh this should be removed like it was in this PR -- https://github.com/ethereum-optimism/optimism-monorepo/pull/173